feature-libs/product-configurator/rulebased/styles/_configurator-attribute-checkbox-list.scss

```diff
@@ -1,3 +1,13 @@
 %cx-configurator-attribute-checkbox-list {
   @include cx-configurator-attribute-type();
+
+  @include forVersion(3.1) {
```

Comment:
There are `@include forVersion(3.1)` in many scss files. Please be careful of the version.

Reply:
picked

Reply:
There is a task #11238 for it

Reply:
Done. Adjusted to 3.3